build.sh

```diff
@@ -2,12 +2,13 @@
 
 set -e
 
+# configuration
 KERNEL_VERSION_RPI1=4.4.0-1-rpi
 KERNEL_VERSION_RPI2=4.4.0-1-rpi2
 
-INSTALL_MODULES="kernel/fs/btrfs/btrfs.ko"
-INSTALL_MODULES="$INSTALL_MODULES kernel/drivers/scsi/sg.ko"
-INSTALL_MODULES="$INSTALL_MODULES kernel/net/ipv6/ipv6.ko"
+INSTALL_MODULES=("kernel/fs/btrfs/btrfs.ko")
```

Comment:
the use of bash arrays requires the use of "newer" versions of bash (3.1 and higher, I think)...
while I personally don't have an issue with this, I do know of cases where this will break for the user

and true, bash 3.1 isn't really that new, but still...